R/Module-Characterization.R

```diff
@@ -36,28 +34,13 @@ CharacterizationModule <- R6::R6Class(
         cdmDatabaseSchema = jobContext$moduleExecutionSettings$cdmDatabaseSchema,
         characterizationSettings = jobContext$settings,
         databaseId = jobContext$moduleExecutionSettings$databaseId,
-        saveDirectory = workFolder,
-        tablePrefix = self$tablePrefix
-      )
-
-      # Export the results
-      rlang::inform("Export data to csv files")
-
-      sqliteConnectionDetails <- DatabaseConnector::createConnectionDetails(
-        dbms = "sqlite",
-        server = file.path(workFolder, "sqliteCharacterization", "sqlite.sqlite")
-      )
-
-      # get the result location folder
-      resultsFolder <- jobContext$moduleExecutionSettings$resultsSubFolder
-
-      Characterization::exportDatabaseToCsv(
-        connectionDetails = sqliteConnectionDetails,
-        resultSchema = "main",
-        tempEmulationSchema = NULL,
-        tablePrefix = self$tablePrefix,
-        filePrefix = self$tablePrefix,
-        saveDirectory = resultsFolder
+        outputDirectory = jobContext$moduleExecutionSettings$resultsSubFolder,
+        executionPath = workFolder,
+        csvFilePrefix = self$tablePrefix,
+        minCellCount =  jobContext$moduleExecutionSettings$minCellCount,
+        minCharacterizationMean = 0.01,
```

Comment:
I believe this should be `minCharacterizationMean = jobContext$moduleExecutionSettings$minCharacterizationMean`

Reply:
those are different - Char has a minCellCount that is an integer - this is applied to all results to remove counts less than minCellCount and results linked to those counts.  minCharacterizationMean is just used by FeatureExtraction to only keep sufficiently common features.

Reply:
But I see that the Characterization module supports defining the `minCharacterizationMean` as shown [here](https://github.com/OHDSI/Strategus/pull/152/files#diff-ea846a7369d5979e678c6f056f80bda93b9db864a6217f4efb92a755148947b5R129) so I thought that's what you'd want to pass vs. hard-coding this value?